src/Raven.Server/Indexing/VoronIndexOutput.cs

```diff
@@ -37,6 +44,7 @@ public VoronIndexOutput(StorageEnvironmentOptions options, string name, Transact
         public override void FlushBuffer(byte[] b, int offset, int len)
         {
             _file.Write(b, offset, len);
+            TotalWritten += len;
```

Comment:
I'm wondering if we can do better here... not sure if this is enough to measure the allocations, but I'm not sure what else could we do?

Reply:
The total value is the same as _file.Length (which is more expensive)